### PR TITLE
fixes #12569 - update left list w/ as changes are saved

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

**Comment by thomasmckay** [comment]
[test]

**Comment by thomasmckay** [comment]
I don't know what the javascript failure is. 'grunt ci' runs clean for me locally.

**Comment by thomasmckay** [comment]
@waldenraines - Mind reviewing?

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details.controller.js [hunk 1]
```diff
@@ -86,9 +86,10 @@ angular.module('Bastion.content-hosts').controller('ContentHostDetailsController
                 deferred.resolve(response);
                 $scope.host = response;
                 $scope.successMessages.push(translate('Save Successful.'));
+                $scope.table.replaceRow(response);
             }, function (response) {
                 deferred.reject(response);
-                _.each(response.data.errors, function (errorMessage) {
+                _.each(response.data.error.full_messages, function (errorMessage) {
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details.controller.js]: Why the change to how we handle errors?

**Comment by beav** [comment]
I tested this but kept hitting http://projects.theforeman.org/issues/15209 when trying to set up a repro case. I will retest once that issue is resolved.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details.controller.js (hunk 1 above)
Comment (thomasmckay) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details.controller.js]: The error structure returned by saving a host is different than saving an old system.

Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details.controller.js]: 👍

**Comment by johnpmitsch** [comment]
I'm not seeing errors being reported properly using this diff

```
diff --git a/app/lib/katello/resources/candlepin.rb b/app/lib/katello/resources/candlepin.rb
index 0f31718..816f303 100644
--- a/app/lib/katello/resources/candlepin.rb
+++ b/app/lib/katello/resources/candlepin.rb
@@ -174,6 +174,7 @@ module Katello
           end
 
           def consume_entitlement(uuid, pool, quantity = nil)
+            fail "hi"
             uri = join_path(path(uuid), 'entitlements') + "?pool=#{pool}"
             uri += "&quantity=#{quantity}" if quantity && quantity > 0
             response = self.post(uri, "", self.default_headers).body
```

but this is also an issue in master so I opened a new issue here http://projects.theforeman.org/issues/16473

ACK
